frontend/packages/noobaa-storage-plugin/src/components/status-card/statuses.tsx

```diff
@@ -39,7 +39,7 @@ export const getNooBaaState: GetObjectServiceStatus = (
   const noobaaPhase = _.get(k8sResponse, 'data[0].status.phase');
   const unhealthyBucketsRatio = unhealthyBuckets / buckets;
 
-  if (hasLoadError) {
+  if (hasLoadError || !(buckets && unhealthyBuckets && pools && unhealthyPools && noobaaPhase)) {
```

Comment:
Its better to have one function separately to check the conditions, for better readability.

Reply:
Can you please tell what do you mean by separate function?
I did not understand.
We have this function for calculating status. For readability, I moved it to a const now.

Reply:
thats what I was referring to.

Reply:
Okay got it! Thanks.